README.md

```diff
@@ -40,7 +40,8 @@ Config for the application env can be one of the following:
       {app_name, "my-app"}
       {type, syslog_udp},
 	  {syslog_udp_host, {127, 0, 0, 1}},
-	  {syslog_udp_port, 514}
+	  {syslog_udp_port, 514},
+          {syslog_udp_facility, 10}
```

Comment:
For some reason the README seems to have tabs in it which means that this new line is showing up as misaligned. I'll merge your PR and remove the tabs in a separate commit.